### ISSUE TITLE
Different SROCC results for individual frame rates on the LIVE-YT-HFR dataset

### ISSUE THREAD

**Issue by qaohv** [pavancm/GREED#4]
Hello, Pavan.

I tried to reproduce results for GREED-bior2.2 model for individual frame rates on the LIVE-YT-HFR dataset (Table II from article "ST-GREED: Space-Time Generalized Entropic Differences for Frame Rate Dependent Video Quality). To do this I used code from repo and pretrained on LIVE-YT-HFR database model from repository ("model_params/bior22.model" and "model_params/bior22_params.mat"). 
I predicted scores and calculated SROCC for each frame rate group, results shown in Table 1.

Table 1. SROCC for individual frame rates on the LIVE-YT-HFR dataset

|  | 24 | 30 | 60 | 82 | 98 | 120 | Overall |
|:---:|:---:|:---:|:---:|:---:|:---:|:---:|---|
| GREED bior2.2 (article) | 0.7268 | 0.7018 | 0.7321 | 0.8179 | 0.8643 | 0.8881 | 0.8822 |
| GREED bior2.2 (our) | 0.7516 | 0.7623 | 0.8244 | 0.8826 | 0.8924 | 0.9245 | 0.9172 |

According the Table 1. model from repository shows better results then reported in a article. I guess it happened bacause I used for measurements model trained on full dataset.  Please, clarify which model did you use (trained on full dataset or only part of them) to predict scores for experiment on individual frame rates?

**Comment by pavancm** [comment]
Yes, the released version is trained on the whole dataset. So if you again test on them, there will be overlap in testing and training sets leading to increased correlations. In order to reproduce the values reported in the article you need to divide the dataset into train and test sets, and calculate performance only on the test set. Also the division has to be done according to content (please refer to the article to know more about this division).

**Comment by qaohv** [comment]
Thanks for clarification!
